sematic/cli/start.py

```diff
@@ -32,4 +32,12 @@ def start():
 
         os._exit(0)
 
-    run_wsgi(False)
+    # This enables us to use websockets and standard HTTP
+    # requests in the same server locally, which is what
+    # we want. If you try to use Gunicorn to do this,
+    # gevent will complain about not monkey patching
+    # early enough, unless you have the gevent monkey
+    # patch applied VERY early (like user/sitecustomize).
+    # monkey patching:
+    # https://github.com/gevent/gevent/issues/1235
```

Comment:
Please either move or copy this comment to the `socketio.run(` line inside `run_socketio`.